streamz/core.py

```diff
@@ -304,6 +304,27 @@ def union(self, *others):
         """
         return union(children=(self,) + others)
 
+    def pluck(self, pick):
+        """ Select elements from a tuple in the stream.
```

Comment:
I recommend replacing "from a tuple" with "from elements" .  This will work with any indexable, including dicts.